editor/src/core/tailwind/tailwind-compilation.ts

```diff
@@ -16,6 +16,30 @@ import type { RequireFn } from '../shared/npm-dependency-types'
 import { TailwindConfigPath } from './tailwind-config'
 import { ElementsToRerenderGLOBAL } from '../../components/canvas/ui-jsx-canvas'
 import { isFeatureEnabled } from '../../utils/feature-switches'
+import type { Config } from 'tailwindcss/types/config'
+import type { EditorState } from '../../components/editor/store/editor-state'
+import { createRequireFn } from '../property-controls/property-controls-local'
+
+const LatestConfig: { current: { code: string; config: Config } | null } = { current: null }
+export function getTailwindConfig(editorState: EditorState): Config | null {
```

Comment:
maybe putting cached in the name helps us remember that there is memoization happening here
```suggestion
export function getTailwindConfigCached(editorState: EditorState): Config | null {
```

Reply:
https://github.com/concrete-utopia/utopia/pull/6500/commits/df49f33e57e08514accf914bed2bda2d45f1d20e